src/release/main.d

```diff
@@ -383,6 +387,37 @@ ActionList prepareMajorRelease ( ref HTTPConnection con, ref Repository repo,
 
     auto current_branch = SemVerBranch(getCurrentBranch());
 
+    // Make sure no merges are between the last major and this
+    if (current_branch.major > 1)
+    {
+        import release.shellHelper;
+
+        auto previous_major_branch = current_branch;
+        previous_major_branch.major--;
+
+        auto merges = cmd(format("git log --oneline --merges %s..%s",
+                                 previous_major_branch, current_branch));
+
+        if (merges.length > 0)
+        {
+            import std.stdio;
+            writefln("Found merges between this and the last major "~
```

Comment:
Better to add it when it becomes necessary than try to make everything configurable in my opinion.